workbench/views.py

```diff
@@ -80,14 +79,20 @@ def show_scenario(request, scenario_id, view_name='student_view', template='work
     })
 
 
-def user_list(request):
+def user_list(_request):
```

Comment:
To explicitly address your follow-up point, I don't think we need to be concerned; this is a Django View, not intended to be called directly by developers.